### DISCUSSION TITLE
在使用VcOverlayHtml组件时的问题

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#627] [Q&A]
使用VcOverlayHtml组件在地图上绘制html元素后，如何使鼠标放在html元素上滚动时，由cesium地图响应滚动事件对地图进行放大缩小？现在似乎鼠标等事件都被VcOverlayHtml拦截了，viewer接受不到事件对地图进行控制了？能否单独指定例如鼠标滚动事件不被VcOverlayHtml监听？

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/overlays/vc-overlay-html

我这边从在线文档测试默认就是这种表现啊。

**Reply by zouyaoji** [reply]
你自己给你的元素加一下css
pointer-events: none;
